### PR TITLE
Timur katimullin tasks 91 100

### PR DESCRIPTION
**Pull request by linkouth**
Adds 91-100 tasks solutions.

### CODE REVIEW

File: tasks/91_make-a-person/make-a-person.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  /**
+   * setFullName - set full name (first and last)
+   * @param {String} firstAndLast
+   * @return {void}
+   */
+  setFullName(firstAndLast) {
+    const names = firstAndLast.split(' ');
+    this.setFirstName(names[0]);
```
Comment (vovcyan) [inline tasks/91_make-a-person/make-a-person.js]: Please use ES6 features

File: tasks/95_fat-fingers/fat-fingers.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Freddy has a really fat left pinky finger, and every time Freddy
+ * tries to type an A, he accidentally hits the CapsLock key
+ * @example
+ * // return 'The quick brown fox jumps over the lZY DOG.'
+ * fatFingers("The quick brown fox jumps over the lazy dog.");
+ * @param {String} str
+ * @return {String}
+ */
+export function fatFingers(str) {
+  if (str.length === 0) {
```
Comment (vovcyan) [inline tasks/95_fat-fingers/fat-fingers.js]: !str.length

File: tasks/98_linked-list/linked-list.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+  /**
+   * pop - pop element from the end of the list
+   * @example
+   * // return 1
+   * list.pop();
+   * @return {*}
+   */
+  pop() {
+    if (this.tail === null) {
```
Comment (vovcyan) [inline tasks/98_linked-list/linked-list.js]: !this.tail

File: tasks/99_stack/Stack.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  /**
+   * pop - pop node from the top of the stack
+   * @example
+   * stack.pop();
+   * @return {*}
+   */
+  pop() {
+    if (this.last === null) {
```
Comment (vovcyan) [inline tasks/99_stack/Stack.js]: The same

File: tasks/99_stack/Stack.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  /**
+   * isEmpty - returns true if stack is empty, else false
+   * @example
+   * stack.isEmpty();
+   * @return {Boolean}
+   */
+  isEmpty() {
+    return this.size === 0;
```
Comment (vovcyan) [inline tasks/99_stack/Stack.js]: !!this.size

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by akimovaa — APPROVED** [review]

File: tasks/91_make-a-person/make-a-person.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  getFirstName() {
+    return this.firstName;
+  }
+
+  /**
+   * getLastName - returns last name
+   * @return {String}
+   */
+  getLastName() {
+    return this.lastName;
```
Comment (KalachevDev) [inline tasks/91_make-a-person/make-a-person.js:26]: It seem you have to implement it other way. You need to keep it as private

File: tasks/95_fat-fingers/fat-fingers.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * isUpperCase - returns true if char is in upper case, else false
+ * @example
+ * // return true
+ * isUpperCase('B');
+ * @param {String} char
+ * @return {Boolean}
+ */
+const isUpperCase = (char) => {
+  return char >= 'A' && char <= 'Z';
```
Comment (KalachevDev) [inline tasks/95_fat-fingers/fat-fingers.js:42]: you could use regexp here

**Review by vovcyan — APPROVED** [review]

**Pull request merged**